### DISCUSSION TITLE
Transferring nixpkgs-xr to nix-community organization

### DISCUSSION THREAD

**Discussion by Scrumplex** [nix-community/infra#1077] [Q&A] [answered]
Hey there!

I am the author of <https://github.com/Scrumplex/nixpkgs-xr>, which is a self-updating Nixpkgs overlay for XR related packages. This is basically like <https://github.com/nix-community/nixpkgs-wayland> but specifically for XR applications and tools.

As it is right now, it only features [Monado](https://monado.freedesktop.org) and [OpenComposite](https://gitlab.com/znixian/OpenOVR), but I plan to add more once there are more suitable packages in Nixpkgs.
Some candidates include https://github.com/yshui/index_camera_passthrough https://github.com/SlimeVR/SlimeVR-OpenVR-Driver but probably more as the FOSS VR/XR ecosystem matures

My main motivation for moving this to nix-community is to allow more collaboration and having access to continuous integration.

**Comment by Mic92** [comment, marked as answer]
It would be nice to have some usage example in the README. The onboarding workflow usually works like this:

1. We invite you to nix-community
2. You transfer the repository
3. We give you your admins rights to the repository back and create a team where you can invite other people in to work on the project.

**Reply by Scrumplex** [reply]
I have added a usage example to the README. Admittedly, it is still pretty bare.

Regarding the onboarding, I was reading https://nix-community.org/contact/, and it seemed like GH Discussions were a good place to start ^^
